AdminApi/src/AdminApi/ClientApp/src/app/components/identity/identity-details/deletion-processes/dp-audit-logs/dp-audit-logs-details/dp-audit-logs-details/dp-audit-logs-details.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, OnInit } from "@angular/core";
+import { MatSnackBar } from "@angular/material/snack-bar";
+import { ActivatedRoute } from "@angular/router";
+import { DeletionProcessAuditLog, IdentityService } from "src/app/services/identity-service/identity.service";
+import { HttpResponseEnvelope } from "src/app/utils/http-response-envelope";
+
+@Component({
+    selector: "app-dp-audit-logs-details",
+    templateUrl: "./dp-audit-logs-details.component.html",
+    styleUrls: ["./dp-audit-logs-details.component.css"]
+})
+export class DeletionProcessAuditLogsDetailsComponent implements OnInit {
+    public identityDeletionProcessID: string;
+    public identityAddress: string;
+
+    public header: string;
+    public headerDeletionProcessAuditLog: string;
+    public headerDeletionProcessAuditLogDescription: string;
+
+    public loading: boolean;
+    public deletionProcessesAuditLogTableDisplayedColumns: string[];
+
+    public identityDeletionProcessAuditLogs: DeletionProcessAuditLog[] = [];
+
+    public constructor(
+        private readonly identityService: IdentityService,
+        private readonly snackBar: MatSnackBar,
+        private readonly activatedRoute: ActivatedRoute
+    ) {
+        this.header = "Deletion Process Details";
+        this.headerDeletionProcessAuditLog = "Deletion Process Audit Logs";
+        this.headerDeletionProcessAuditLogDescription = "View deletion process audit logs for Identity.";
+        this.loading = false;
+        this.deletionProcessesAuditLogTableDisplayedColumns = ["id", "createdAt", "message", "oldStatus", "newStatus"];
+
+        this.identityDeletionProcessID = "";
+        this.identityAddress = "";
+    }
+
+    public ngOnInit(): void {
+        this.activatedRoute.params.subscribe((params) => {
+            this.identityAddress = params["address"];
+            this.loadIdentityDeletionProcessAuditLogs();
+        });
+    }
+
+    private loadIdentityDeletionProcessAuditLogs(): void {
+        this.loading = true;
+        this.identityService.getDeletionProcessAuditLogsOfIdentity(this.identityAddress.trim()).subscribe({
+            next: (data: HttpResponseEnvelope<DeletionProcessAuditLog[]>) => {
+                this.identityDeletionProcessAuditLogs = data.result.sort((a, b) => new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime());
```

Comment:
Can you please remove the sorting? We now sort them in ascending order in the backend. I think that makes more sense. And your sorting here breaks the backend sorting ;)

Same for the deletion process details view.

Reply:
Changes made. In Deletion Process details view I am not using any sorting to sort the audit logs.